menu.kando.Kando.yml

```diff
@@ -0,0 +1,35 @@
+id: menu.kando.Kando
+runtime: org.freedesktop.Platform
+runtime-version: "24.08"
+sdk: org.freedesktop.Sdk
+command: AppRun
+finish-args:
+  - --share=ipc
+  - --device=dri
+  - --socket=x11
+  - --socket=wayland
```

Comment:
no use only x11, those wanting to opt in can set it themselves.

Reply:
Done.